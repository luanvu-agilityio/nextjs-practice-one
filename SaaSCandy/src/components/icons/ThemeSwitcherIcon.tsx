const ThemeSwitcherIcon = (props: React.SVGProps<SVGSVGElement>) => (
  <svg
    width='40'
    height='40'
    viewBox='0 0 40 40'
    fill='none'
    xmlns='http://www.w3.org/2000/svg'
    {...props}
  >
    <rect width='40' height='40' rx='20' fill='white' />
    <path
      d='M20 30C25.523 30 30 25.523 30 20C30 19.537 29.306 19.46 29.067 19.857C28.5572 20.7013 27.862 21.4186 27.034 21.9545C26.206 22.4903 25.2669 22.8307 24.2878 22.9499C23.3088 23.0691 22.3154 22.9639 21.383 22.6423C20.4507 22.3207 19.6037 21.7911 18.9063 21.0937C18.2089 20.3963 17.6793 19.5493 17.3577 18.617C17.0361 17.6846 16.9309 16.6912 17.0501 15.7122C17.1693 14.7331 17.5097 13.794 18.0455 12.966C18.5814 12.138 19.2987 11.4428 20.143 10.933C20.54 10.693 20.463 10 20 10C14.477 10 10 14.477 10 20C10 25.523 14.477 30 20 30Z'
      fill='black'
    />
  </svg>
);
export default ThemeSwitcherIcon;
