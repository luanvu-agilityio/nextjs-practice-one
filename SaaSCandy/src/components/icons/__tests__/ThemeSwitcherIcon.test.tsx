import { render } from '@testing-library/react';
import ThemeSwitcherIcon from '../ThemeSwitcherIcon';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<ThemeSwitcherIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
