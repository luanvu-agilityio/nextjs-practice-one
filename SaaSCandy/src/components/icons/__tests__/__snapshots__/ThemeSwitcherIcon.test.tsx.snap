// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon > matches snapshot 1`] = `
<svg
  fill="none"
  height="40"
  viewBox="0 0 40 40"
  width="40"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    fill="white"
    height="40"
    rx="20"
    width="40"
  />
  <path
    d="M20 30C25.523 30 30 25.523 30 20C30 19.537 29.306 19.46 29.067 19.857C28.5572 20.7013 27.862 21.4186 27.034 21.9545C26.206 22.4903 25.2669 22.8307 24.2878 22.9499C23.3088 23.0691 22.3154 22.9639 21.383 22.6423C20.4507 22.3207 19.6037 21.7911 18.9063 21.0937C18.2089 20.3963 17.6793 19.5493 17.3577 18.617C17.0361 17.6846 16.9309 16.6912 17.0501 15.7122C17.1693 14.7331 17.5097 13.794 18.0455 12.966C18.5814 12.138 19.2987 11.4428 20.143 10.933C20.54 10.693 20.463 10 20 10C14.477 10 10 14.477 10 20C10 25.523 14.477 30 20 30Z"
    fill="black"
  />
</svg>
`;

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="40"
  viewBox="0 0 40 40"
  width="40"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    fill="white"
    height="40"
    rx="20"
    width="40"
  />
  <path
    d="M20 30C25.523 30 30 25.523 30 20C30 19.537 29.306 19.46 29.067 19.857C28.5572 20.7013 27.862 21.4186 27.034 21.9545C26.206 22.4903 25.2669 22.8307 24.2878 22.9499C23.3088 23.0691 22.3154 22.9639 21.383 22.6423C20.4507 22.3207 19.6037 21.7911 18.9063 21.0937C18.2089 20.3963 17.6793 19.5493 17.3577 18.617C17.0361 17.6846 16.9309 16.6912 17.0501 15.7122C17.1693 14.7331 17.5097 13.794 18.0455 12.966C18.5814 12.138 19.2987 11.4428 20.143 10.933C20.54 10.693 20.463 10 20 10C14.477 10 10 14.477 10 20C10 25.523 14.477 30 20 30Z"
    fill="black"
  />
</svg>
`;
